src/core/include/mp-units/quantity_point.h

```diff
@@ -182,6 +184,27 @@ class quantity_point {
     return make_quantity_point<PO>(quantity_ref_from(PO).force_in(U{}));
   }
 
+  // conversion operators
+  template<QuantityPointLike QP>
+  [[nodiscard]] explicit(
+    is_specialization_of<decltype(quantity_point_like_traits<QP>::from_quantity(quantity_from_origin_)),
+                         convert_explicitly>) constexpr
+  operator QP() const& noexcept(noexcept(quantity_point_like_traits<QP>::from_quantity(quantity_from_origin_)) &&
+                                std::is_nothrow_copy_constructible_v<rep>)
+  {
+    return quantity_point_like_traits<QP>::from_quantity(quantity_from_origin_).value;
+  }
+
+  template<QuantityPointLike QP>
+  [[nodiscard]] explicit(
+    is_specialization_of<decltype(quantity_point_like_traits<QP>::from_quantity(quantity_from_origin_)),
+                         convert_explicitly>) constexpr
+  operator QP() const&& noexcept(noexcept(quantity_point_like_traits<QP>::from_quantity(quantity_from_origin_)) &&
+                                 std::is_nothrow_copy_constructible_v<rep>)
```

Comment:
Either apply this suggestion, or
remove this overload (the other one has equivalent effects).
```suggestion
  operator QP() && noexcept(noexcept(quantity_point_like_traits<QP>::from_quantity(quantity_from_origin_)) &&
                                 std::is_nothrow_move_constructible_v<rep>)
```